common/ConfigUtil.cpp

```diff
@@ -104,7 +104,7 @@ static const std::map<std::string, std::string> DefAppConfig = {
     { "logging.file.property[0][@name]", "path" },
     { "logging.file.property[1]", "never" },
     { "logging.file.property[1][@name]", "rotation" },
-    { "logging.file.property[2]", "false" },
+    { "logging.file.property[2]", "timestamp" },
```

Comment:
as an aside, is it possible to reorder these to have the name property before its value, reading it backwards "timestamp" as value for "archive" is a pain and I'm not the only one who has occasionally associated the value with the preceding name  :-)

Reply:
Yes, agreed. It annoys me as well. Of course I did it with a tool, so it's lexicographical sorting.

Will reverse them. Thanks.